ci/test-conformance-gke.sh

```diff
@@ -89,6 +90,10 @@ case $key in
     USER_EMAIL="$2"
     shift 2
     ;;
+    --credential)
```

Comment:
"credential" is a word with a general meaning. Maybe use more specific words for this argument, such as `--built-in-credential` or `--integrated-credential`. Same comment for ACCOUNT_CREDENTIAL.

Reply:
Updated.